TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/command_runner.py

```diff
@@ -0,0 +1,70 @@
+import subprocess
+import os
+import logging
+from typing import Any, Dict, List, Optional
+
+from console_link.models.command_result import CommandResult
+
+
+logger = logging.getLogger(__name__)
+
+FlagOnlyArgument = None
+
+
+class CommandRunner:
+    def __init__(self, command_root: str, command_args: Dict[str, Any], sensitive_fields: Optional[List[str]] = None,
+                 run_as_detatched: bool = False, log_file: Optional[str] = None):
+        self.command_args = command_args
+        self.command = [command_root]
+        for key, value in command_args.items():
+            self.command.append(key)
+            if value is not FlagOnlyArgument:
+                if type(value) is not str:
+                    value = str(value)
+                self.command.append(value)
```

Comment:
nit: do we need to add \"? Can the values contain spaces? Do we have a test that tests for this?

password: my password